cli/src/pcluster/schemas/imagebuilder_schema.py

```diff
@@ -46,7 +47,11 @@ def make_resource(self, data, **kwargs):
 class ImageSchema(BaseSchema):
     """Represent the schema of the ImageBuilder Image."""
 
-    name = fields.Str(validate=validate.Regexp(r"^[-_A-Za-z-0-9][-_A-Za-z0-9 ]{1,126}[-_A-Za-z-0-9]$"), required=True)
+    name = fields.Str(
+        validate=validate.Regexp(r"^[-_A-Za-z-0-9][-_A-Za-z0-9 ]{1,126}[-_A-Za-z-0-9]$")
```

Comment:
I am not sure this regex pattern already excludes the pattern `".*{{\s*imagebuilder:buildDate\s*}}.*"`
So condition `if not AMI_NAME_REQUIRED_SUBSTRING_PATTERN.match(self.imagebuild.image.name)` may always be true.

Reply:
correct. Removed `AMI_NAME_REQUIRED_SUBSTRING_PATTERN`